### ISSUE TITLE
[BUG] 小米等国产设备上，所有文字字体变粗

### ISSUE THREAD

**Issue by ppolariss** [DanXi-Dev/DanXi#444]
**复现步骤**
复现该 Bug 的步骤：
1. 安装最新apk后，对照旧版本可以直接看出

**截图**
之前
![d9298186418b5c385e132192888228e](https://github.com/user-attachments/assets/3214b8ef-b705-45c0-9ef8-465c6e179c98)
之后
![3a5767e0f7ad8deb569c1f860102a0b](https://github.com/user-attachments/assets/a869b16e-8901-403b-8668-fa4e377b3c51)

**系统信息**
 - 设备型号：Xiaomi13
 - 系统版本：HyperOS1.0.17.0 Android14
 - 旦挞校园助手版本（见「设置-关于本应用」）：1.4.4 build 341

**Comment by w568w** [comment]
已在 https://github.com/flutter/flutter/issues/153748 中修复。

**Comment by w568w** [comment]
目前出现该问题的设备均为小米 + MIUI + Mi-Sans 字体。尝试在 master 频道的 Flutter 上编译，问题似乎没有得到修复。情况和上面的 issue 有差别。

可以确定的是，该问题在 Flutter 3.24.0 引入。

**Comment by w568w** [comment]
也需查看 <https://github.com/flutter/flutter/issues/154166> 和 <https://github.com/flutter/flutter/issues/105014>。

**Comment by kooWZ** [comment]
@w568w @ppolariss  fixed in 1.4.6?

**Comment by w568w** [comment]
> fixed in 1.4.6?

Yes, it should have been fixed in f6bd1744e5f23b07be00dba45898dfed7c9f27e8. Thanks for notification!
